web/src/app/error/error.module.ts

```diff
@@ -0,0 +1,18 @@
+// Core components
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+// Application component
+import { SharedModule } from '@app/shared/shared.module';
```

Comment:
Here we can use `@shared` instead of `@app/shared`

Reply:
Good spot 👍

Reply:
I just copied that part 😂 😜 - I didn't spot it 🤦‍♂ . I will update

Reply:
Yes it was my mistake :( . I noticed now so I commented